tests/basic_correctness/test_chunked_prefill.py

```diff
@@ -62,3 +77,75 @@ def test_models(
         name_0="hf",
         name_1="vllm",
     )
+
+
+@pytest.mark.parametrize("model", LOG_PROBS_MODELS)
+@pytest.mark.parametrize("kv_cache_dtype", ["fp8", "fp8_e5m2"])
+@pytest.mark.parametrize("max_tokens", [32])
+@pytest.mark.parametrize("chunked_prefill_token_size", [1, 4, 16])
+@pytest.mark.parametrize("enforce_eager", [False, True])
+# NOTE: Increasing this in this suite will fail CI because we currently cannot
+# reset distributed env properly. Use a value > 1 just when you test.
+@pytest.mark.parametrize("tensor_parallel_size", [1])
+def test_models_log_probs(
+    vllm_runner,
+    example_prompts,
+    model: str,
+    kv_cache_dtype: str,
+    max_tokens: int,
+    chunked_prefill_token_size: int,
+    enforce_eager: bool,
+    tensor_parallel_size: int,
+) -> None:
+    """
+    Only checks log probs match between chunked-prefill and
+    non-chunked-prefill version of vLLM model runner.
+    
+    This test is used when there is discrepancy in kernels
+    / numerics (e.g. when using lower-precision types like FP8).
+    """
+    if kv_cache_dtype == "fp8_e5m2" and model not in E5M2_KV_MODELS:
+        pytest.skip(
+            f"{model} not in {E5M2_KV_MODELS} requiring {kv_cache_dtype}")
+    elif "fp8" in kv_cache_dtype and model not in E4M3_KV_MODELS:
+        pytest.skip(
+            f"{model} not in {E4M3_KV_MODELS} requiring {kv_cache_dtype}")
+
+    NUM_LOG_PROBS = 8
+    NUM_OUTPUT_TOKENS = 4
```

Comment:
Why you need to compute 32 max new tokens but only check the first 4? You should just configure the max_tokens to be the number of tokens you want to check.